app/gateway/2.8.x/plan-and-deploy/security/secrets-management/backends-overview.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Supported Vault Backends
+beta: true
+---
+
+{:.warning}
+> **Beta warning:**
+> <br>
+> Secrets Rotation is currently not supported in this version.
+
+The following Vault implementations are supported:
+
+|                  | [AWS SecretsManager](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/backends/aws-sm) | [Hashicorp Vault](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/backends/hashicorp-vault) | [Environment Variable](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/backends/env) |
```

Comment:
```suggestion
|                  | [AWS Secrets Manager](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/backends/aws-sm) | [Hashicorp Vault](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/backends/hashicorp-vault) | [Environment Variables](/gateway/{{page.kong_version}}/plan-and-deploy/security/secrets-management/backends/env) |
```

I would also switch the `Get`/`Set`/`Tier` to top, because is is more likely that new vaults are added, and then it makes it easier to expand the table vertically rather than horizontally.